src/app.js

```diff
@@ -6,8 +6,166 @@ import _ from 'underscore';
+      }
+    });
+  }
+};
+
+const reportStatus = function reportStatus(status, field, problem) {
+  console.log('in reportStatus function');
+  console.log(`Reporting ${ status } status: ${ field } problem: ${problem}`);
+  const errorSpanElement = $(`#form-${field}`)
+  errorSpanElement.html('');
+  const generatedHTML = $(statusTemplate({'problem': problem }));
+  errorSpanElement.append(generatedHTML);
+};
+
+const clearFormMessages = function clearFormMessages() {
+  $('#add-trip-form span').html('')
+};
+
+const handleValidationFailures = function handleValidationFailures(errors) {
+  console.log('in handleValidationFailures function');
+  for (let field in errors) {
+    for (let problem of errors[field]) {
+      reportStatus('error', field,  problem);
+    }
+  }
+};
+
+const addTripHandler = function(event) {
+  console.log('addTripHandler entered');
+  event.preventDefault();
+
+  const trip = new Trip(readResFormData());
+  if (!trip.isValid()) {
+    console.log(`Invalid Trip`);
+    handleValidationFailures(trip.validationError);
+    return;
+  }
+
+  tripList.add(trip);
+
+  trip.save({}, {
+    success: (model, response) => {
+      console.log('Successfully saved Trip!');
+      $('#myModal').hide();
+      $(`#add-trip-form input`).val('');
+      // TODO: get line of code above working
+      reportStatus('success', 'Successfully saved trip!');
+    },
+    error: (model, response) => {
+      console.log('Failed to save bookTrip! Server response:');
+      console.log(response);
+
+      tripList.remove(model);
+      console.log(response.responseJSON["errors"]);
+      handleValidationFailures(response.responseJSON["errors"]);
+    },
+  });
+};
+
+const readResFormData = function readResFormData(){
+  const tripData = {};
+  TRIP_FIELDS.forEach((field) => {
+    const inputElement = $(`#add-trip-form input[name="${ field }"]`);
+    const value = inputElement.val();
+
+    if (value != '') {
+      tripData[field] = value;
+    }
+
+  });
+  console.log("Read trip data");
+  console.log(tripData);
+
+  return tripData;
+};
+
+const render = function render(tripList) {
+  const tripTableElement = $('#trip-list');
+
+  tripTableElement.html('');
+
+  tripList.forEach((trip) => {
+    const generatedHTML = $(tripTemplate(trip.attributes));
+    generatedHTML.on('click', (event) => {
+      renderDetails(trip);
+    });
+    tripTableElement.append(generatedHTML);
+  });
+  $('th.sort').removeClass('current-sort-field');
+  $(`th.sort.${ tripList.comparator }`).addClass('current-sort-field');
+};
 
 $(document).ready( () => {
-  $('main').html('<h1>Hello World!</h1>');
+  let modal = $('#myModal');
+
+  detailsTemplate = _.template($('#details-template').html());
+  tripTemplate = _.template($('#trip-template').html());
+  statusTemplate = _.template($('#status-message-template').html());
+
+  tripList.on('update', render);
+
+  tripList.fetch();
+
+  $('#add-trip-form').on('submit', addTripHandler);
```

Comment:
You're adding this event handler twice (check line 159), this results in the trip appearing in the list twice.